routes/doctor.js

```diff
@@ -17,6 +17,32 @@ router.get('/:doctorId/patientArray', async function(req, res) {
   }
 });
 
+router.post('/:doctorId/appointment', async (req, res) => {
```

Comment:
minor detail, but maybe we can rename this route to 'create-appointment' to make it clearer